app/components/degrees_review_component.html.erb

```diff
@@ -2,8 +2,11 @@
   <%= render(SummaryCardComponent, rows: degree_form_rows(degree_form)) do %>
     <header class="app-summary-card__header">
       <h3 class="app-summary-card__title">
-        <%= formatted_degree_title(degree_form.qualification_type, degree_form.subject) %>
+        <%= degree_form.title %>
       </h3>
+      <div class="app-summary-card__actions">
+        <%= link_to t('application_form.degree.delete'), candidate_interface_degrees_destroy_path(degree_form.id), class: 'govuk-link' %>
```

Comment:
nitpick:
you could use:
`candidate_interface_degrees_destroy_path(degree_form)`
instead of
`candidate_interface_degrees_destroy_path(degree_form.id)`

Reply:
I believe you're right, but in this case it didn't work. It strangely couldn't find the `id`.